packages/dux-code-report/index.js

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright IBM Corp. 2022, 2022
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+const args = process.argv.slice(2);
+// const path = require('path');
+const fs = require('fs');
+
+// console.log(args);
+
+const jsRegex = new RegExp(
+  'import {(.*)} from [\'"]@carbon/((ibm-products)|(ibm-cloud-cognitive))[\'"].*',
+  'gi'
+);
+
+const genReport = async () => {
+  // const promises = [];
+
+  console.log('Scanning for use of Carbon for IBM Products\n');
+  console.log(
+    '-----------------------------------------------------------------------------\n'
+  );
+  console.log('Submit the following as part of your DUX report.\n');
```

Comment:
We certainly mentioned how this could be useful for DUX review, but I don't think it helps greatly to limit this to that. I don't think we need explicitly mention DUX anywhere, and I don't think the package name should include it. Perhaps it's a 'component usage report'?